docs/nlu/components.rst

```diff
@@ -951,6 +956,36 @@ EntitySynonymMapper
         When using the ``EntitySynonymMapper`` as part of an NLU pipeline, it will need to be placed
         below any entity extractors in the configuration file.
 
+
+.. _RegexEntityExtractor:
+
+RegexEntityExtractor
+~~~~~~~~~~~~~~~~~~~~~
+
+:Short: Extracts entities using the lookup tables and regexes defined in the training data
+:Outputs: ``entities``
+:Requires: Nothing
+:Description:
+    This component extract entities using the lookup tables and regexes defined in the training data.
+    The component checks if the user message contains an entry of one of the lookup tables or matches one of the
+    regexes. If a match is found, the value is extracted as entity.
+
+    .. note::
+        You cannot use the :ref:`RegexFeaturizer` and the ``RegexEntityExtractor`` in the same pipeline. You need to
+        decide whether you want to use the lookup tables and regexes to create features or to directly extract entities.
+        It is not possible to do both at the same time.
+
+:Configuration:
```

Comment:
I'd add a config to use only lookup or only regex or both. E.g. I can imagine when users want to use only lookup tables

Reply:
Ah ok, so you would do something like this?
```
- name: RegexEntityExtractor
  use_lookup_tables: True
  use_regex_features: False
  lowercase: True
```
Would you then also add the same to the `RegexFeaturizer`?

Reply:
yes, we could do that, but I think it's fine for featurizer to use all

Reply:
Ok. Vincent mentioned a similar point but we weren't sure if the config would become even more complicated with this addition. But I guess we can add this.

Reply:
since we have defaults, this config becomes entirely optional